pkg/traces/instance.go

```diff
@@ -180,14 +198,28 @@ func (i *Instance) buildAndStartPipeline(ctx context.Context, cfg InstanceConfig
 		MeterProvider:  metric.NewNoopMeterProvider(),
 	}
 
+	// start extensions
+	i.extensions, err = extensions.Build(settings, appinfo, otelConfig, factories.Extensions)
+	if err != nil {
+		i.logger.Error(fmt.Sprintf("failed to build extensions:%s", err.Error()))
+		return fmt.Errorf("failed to create extensions builder: %w", err)
+	}
+	i.logger.Info(fmt.Sprintf("starting extensions:%+v", i.extensions))
```

Comment:
I would drop this log and the one in line 219. The collector's logger already handles logging during start-up.

Also, they will print the pointers to the extensions/exporters, which isn't very helpful.

Reply:
17596c13